src/libs/actions/Report.js

```diff
@@ -500,9 +500,12 @@ function fetchChatReports() {
  * Get the actions of a report
  *
  * @param {Number} reportID
+ * @param {Number} offset
+ * @returns {Promise}
  */
-function fetchActions(reportID) {
-    API.Report_GetHistory({reportID})
+function fetchActions(reportID, offset) {
+    const reportActionsOffset = _.isNumber(offset) ? offset : Math.max(reportMaxSequenceNumbers[reportID] - CONST.REPORT.REPORT_ACTIONS_LIMIT, 0);
```

Comment:
what do we pass on first load? 0?

Reply:
Also we are assuming `reportMaxSequenceNumbers` has a value... I think it's possible for it to have no value at all which would do something like:

```js
Math.max(undefined - 50, 0) // NaN;
```

Reply:
😂 

<img width="221" alt="Screen Shot 2021-03-01 at 9 20 07 AM" src="https://user-images.githubusercontent.com/32969087/109547352-5d49ac80-7a6f-11eb-993c-e69d2438ce8b.png">